### DISCUSSION TITLE
Essai De Classes

### DISCUSSION THREAD

**Discussion by LaMangousteRose** [Crownicles/Crownicles#2359] [Nouvelles idées]
Ce serait bien si on ajoutait une commande comme par exemple "/fight trial :true", puis on réagit sur la réaction du palier de classe que l'on veut choisir, après on réagit sur la réaction de la classe du palier que l'on veut choisir.
Après cela, le bot nous fait fight un boss du jeu chosi au hasard.

Ca servira à essayer de nouvelles classes sans attendre 1 mois pour les utiliser.

**Comment by BananePlantain** [comment]
ca risque d'être long avant d'arriver dans le jeu,en attendant passe testeur 👀

**Comment by LaMangousteRose** [comment]
Ca sert à quoi testeur??? 👀 
Hehe oui je suis un noob

**Comment by 0eddo** [comment]
en gros c un rôle sur le discord où tu testes les prochaines maj etc (je fais vrmt grosso modo)
tu peux demander à devenir testeur à doctor c'est lui qui gère il me semble :)
